### PR TITLE
Replace accidentally-removed SimpliSafe notification docs

### PR DESCRIPTION
**Pull request by bachya**
## Proposed change

I was too exuberant in https://github.com/home-assistant/home-assistant.io/pull/17643 and removed documentation that isn't actually affected by the removal of the SimpliSafe websocket. This PR replaces it.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: N/A
- Link to parent pull request in the Brands repository: N/A
- This PR fixes or closes issue: N/A

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/simplisafe.markdown [hunk 1]
```diff
@@ -92,66 +92,6 @@ For any property denoting a volume, the following values should be used:
 
 ## Events
 
-### `SIMPLISAFE_EVENT`
```
Comment (MartinHjelmare) [inline source/_integrations/simplisafe.markdown]: In the old PR that targeted `next` branch we removed this text already. Hasn't that PR gone out to `current` yet?

Comment (bachya) [reply source/_integrations/simplisafe.markdown]: It doesn't look like `next` has been merged into `current`. The live website still shows the old docs:

image

Comment (MartinHjelmare) [reply source/_integrations/simplisafe.markdown]: Maybe we should have tagged #17643 for the release? The missing tag could be the reason it wasn't cherry picked for the docs rc branch.

@frenck what is the best approach at this point?

Comment (frenck) [reply source/_integrations/simplisafe.markdown]: I have manually cherry-picked #17643 to `current`.

@bachya This PR, of course, now causes a merge conflict. Anything in this PR that still needs to go into current?

Comment (bachya) [reply source/_integrations/simplisafe.markdown]: @frenck Rebased. 👍🏻

**Review by frenck — APPROVED** [review]
Thanks, @bachya 👍

**Pull request merged**